docs/docs/migration-guide.mdx

```diff
@@ -437,6 +437,41 @@ response_selector_output.get("default")
                         .get("intent_response_key")
 ```
 
+### Core Featurization
+
+* `LabelTokenizerSingleStateFeaturizer` is deprecated and will be removed in the future.
+  Please use `SingleStateFeaturizer` with appropriate NLU pipeline.
+  To replicate `LabelTokenizerSingleStateFeaturizer` functionality,
+  add a `Tokenizer` with the option `intent_tokenization_flag: True` and `CountVectorsFeaturizer`
+  to the NLU pipeline. Please update your configuration file.
+
+  For example:
+  ```yaml
+  language: en
+  pipeline:
+    - name: WhitespaceTokenizer
+      intent_tokenization_flag: True
+    - name: CountVectorsFeaturizer
+  ```
+
+  Please train both NLU and Core (using `rasa train`) to use a trained tokenizer and
+  featurizer for Core featurization.
+
+* `BinarySingleStateFeaturizer` is deprecated and will be removed in the future.
+  If you train both NLU and Core (using `rasa train`),
+  use `SingleStateFeaturizer` with NLU pipeline
```

Comment:
Is this featurizer the default? Since it's not in the code snipped you offered below

Reply:
it is not only default, it is the only one. The code snippet below is for NLU pipeline